data/json/npcs/godco/godco_missions.json

```diff
@@ -0,0 +1,1213 @@
+    "end": { "effect": [ { "u_spawn_item": "270win_jsp", "count": 4 }, { "u_spawn_item": "icon", "count": 5 } ] }
+  },
+  {
+    "id": "MISSION_GODCO_RUSSELL_TROPHY3",
+    "type": "mission_definition",
+    "name": { "str": "Trophy Hunt III" },
+    "description": "Bring one last trophy to Russell Connelly.  Harvest a sting from a bee.",
+    "goal": "MGOAL_FIND_ITEM",
+    "difficulty": 3,
+    "value": 0,
+    "start": { "assign_mission_target": { "om_terrain": "hive", "om_special": "Bee Hive", "reveal_radius": 3, "search_range": 280 } },
+    "item": "bee_sting",
+    "count": 1,
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "has_generic_rewards": false,
+    "dialogue": {
+      "describe": "I need your help…",
+      "offer": "It feels like I'm doing this for revenge.  I have one last mission for you.  Ever seen those ginormous bee hives?  Well, I can tell you where the closest one is.  Just kill off one stray bee, harvest its stinger and bring it to me.",
+      "accepted": "I wish you the best of luck, may God guide your path.",
+      "rejected": "This passion of mine may just be getting to me…",
+      "advice": "If I were you, I'd bring a gun and a butcher knife.  I don't think bug spray work anymore.",
+      "inquire": "How is the hunt going, survivor?  Any luck?",
+      "success": "Well, partner, this might just be the end of the road.  I hope you enjoyed it as much as I have.  In the meantime… here's your reward, a handful of icons, that's all I have left.  Spend them all, I think you've deserved it.",
+      "success_lie": "You liar!",
+      "failure": "You give up now?  This country fell apart because no one could find a good man to rely on… might as well give up, I guess."
+    },
+    "end": {
+      "effect": [
+        { "u_spawn_item": "270win_jsp", "count": 8 },
+        { "u_spawn_item": "icon", "count": 8 },
+        { "npc_add_effect": "hunt_godco_russell", "duration": "PERMANENT" }
+      ]
+    }
+  },
+  {
+    "id": "MISSION_GODCO_ZACHARY_KILL_NIGHTMARE",
+    "type": "mission_definition",
+    "name": { "str": "End The Nightmare" },
+    "goal": "MGOAL_KILL_MONSTER",
+    "description": "Kill whatever's keeping Zachary up at night.",
+    "difficulty": 5,
+    "value": 0,
+    "start": {
+      "assign_mission_target": { "om_terrain": "forest", "reveal_radius": 1, "random": false, "search_range": 20, "min_distance": 5, "z": 0 },
+      "update_mapgen": { "place_monster": [ { "monster": "mon_zombie_screecher", "x": 11, "y": 11, "target": true } ] }
+    },
+    "end": {
+      "effect": [
+        { "u_spawn_item": "icon", "count": 12 },
+        { "u_add_var": "zachary_willing_to_follow", "type": "mission", "context": "meeting", "value": "yes" }
+      ]
+    },
+    "origins": [ "ORIGIN_SECONDARY" ],
+    "has_generic_rewards": false,
+    "dialogue": {
+      "describe": "Kill whatever's keeping Zachary up at night.",
+      "offer": ".",
+      "accepted": ".",
+      "rejected": ".",
+      "advice": ".",
```

Comment:
Still needs content. Good idea for a quest though.

Reply:
Those lines are handled via external dialogue and given via external dialogue; the mission isn't supposed to be assigned through the `MISSION_LIST` subject.

Reply:
Definitely agree with some kind of notation that implies if the character sees this it's a bug so that people can know to report it.